config/database.yml

```diff
@@ -1,6 +1,11 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
 development:
-  adapter: postgresql
```

Comment:
This was switched pretty recently to postgres (https://github.com/theworldbright/5c-ride-share/commit/bc5b272fa216a29f62baf7d450c47d5fb92bcf2e). @mattdahl what was the rationale for the switch?